style.css

```diff
@@ -12,9 +8,78 @@
   flex-wrap: wrap;
 }
 
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
 .vertical-line {
   height: 200px;
   width: 3px;
   background: linear-gradient(transparent, #FFD700, transparent);
   margin: 0 30px;
 }
+
+.feature-group {
+  background-color: #EEE;
+  padding: 40px;
+  position: relative;
+}
+
+.features-container {
+  display: flex;
+  justify-content: center;
+  max-width: 1280px;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-grow: 0;
+  margin: 0 50px;
+  max-width: 200px;
+}
+
+.feature-title {
+  color: #111;
+  margin-bottom: 20px;
+}
+
+.feature-description {
+  color: #999;
+}
+
+.text {
+  display: flex;
+  font-weight: bold;
+  flex-wrap: wrap;
+  text-align: center;
+}
+
+.feature-image {
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+
+.feature-group::before {
+  background: url("images/triangle.png");
+  height: 12px;
+  content: "";
+  width: 100%;
+  top: -12px;
+  left: 0;
+  position: absolute;
+}
+
+@media screen and (max-width: 1024px) {
+  .feature-group {
+    flex-direction: column;
+  }
+
+  .main-section {
```

Comment:
nope, just the phone and the line should be removed, not all the section. you should add a class to the phone image and target that one with the vertical line one